ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/JdbcLedgerDao.scala

```diff
@@ -338,6 +338,7 @@ private class JdbcLedgerDao(
                 commandId,
                 None,
                 Some(submissionId),
+                None, // TODO Ledger Metering
```

Comment:
I suggest naming the parameters to CompletionInfo everywhere; there are now a lot of them.

Reply:
Yes, I will do this as part of the PR where I clean up the TODOs.